projects/uitdatabank/docs/search-api/embedding.md

```diff
@@ -308,3 +308,89 @@ Every result in the response will have a `calendarSummary` property with a value
    }
 }
 ```
+
+## Embedding UiTPAS prices
```

Comment:
Hmm I didn't know we would actually document this. We'll also need to add it to the OpenAPI file then. I was under the assumption that we would communicate this to existing integrators as a temporary parameter until they have made any necessary changes, and then just always return the UiTPAS prices and remove the code for the parameter. So it wouldn't exist in x months/years.

Reply:
See also this ticket that I just ran into https://jira.uitdatabank.be/browse/III-4833

Reply:
I think an url is easier for us to communicate and easier to use for integrators than an email (which is more "volatile" in nature)?
In a new commit I emphasized the temporariness of the parameter in a warning block to avoid any confusion

Reply:
Sounds good!